### PR TITLE
multilingual README.md

### PR DESCRIPTION
**Pull request by shizhediao**

### CODE REVIEW

File: README_es.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+    `python ./scripts/convert_llama_weights_to_hf.py --input_dir ${llama-path} --model_size 7B --output_dir ${llama-hf-path}/llama-7b-hf`
+
+3. ¡Listo! Ahora puedes establecer la ruta del checkpoint en `${llama-hf-path}/llama-7b-hf`. ¡Disfrútalo!
+
+
+### 4.2 DeepSpeed Config
+
+Puedes configurar DeepSpeed en configs. Los detalles se pueden consultar en [Configuración de DeepSpeed](https://www.deepspeed.ai/docs/config-json/).
+
+
```
Comment (JohnDuncan5171) [inline README_es.md:254]: Add a new Note for "Our support for Mac OS system is not been thoroughly tested yet."
For more information see [this issue](https://github.com/OptimalScale/LMFlow/issues/64#issuecomment-1495511446)

**Review by JohnDuncan5171 — CHANGES_REQUESTED** [review]
Add a new Note for "Our support for Mac OS system is not been thoroughly tested yet."
For more information see [this issue](https://github.com/OptimalScale/LMFlow/issues/64#issuecomment-1495511446)

**Review by hendrydong — APPROVED** [review]

**Review by JohnDuncan5171 — COMMENTED** [review]
add note for System support

**Comment by hendrydong** [comment]
> add note for System support

Thank you for your suggestion. We will add the system information.

**Pull request merged**